### PR TITLE
Leaderboard 2022

### PR DESCRIPTION
**Pull request by ItsAdityaKSingh**
Added changes for showcasing leaderboard for CSoC 2022.

### CODE REVIEW

File: src/pages/Index.vue [hunk 1]
```diff
@@ -42,6 +42,11 @@ export default {
           description: "Enthusiastic first yearties force COPS to start track based CSOC Dev for both frontend and backend. The Frontend track included good 'ol Web Development and Native Android as well as Cross Platform Flutter Development were the new additions.",
           icon: "otter",
           link: "2021"
+        },
+         {
+          description: "Enthusiastic first yearties force COPS to start track based CSOC Dev for both frontend and backend. The Frontend track included good 'ol Web Development and Native Android as well as Cross Platform Flutter Development were the new additions.",
+          icon: "bug",
```
Comment (king-11) [inline src/pages/Index.vue:48]: We are calling them bugs xD so harsh

**Review by king-11 — APPROVED** [review]

File: src/pages/Index.vue [hunk 2]
```diff
@@ -42,6 +42,11 @@ export default {
           description: "Enthusiastic first yearties force COPS to start track based CSOC Dev for both frontend and backend. The Frontend track included good 'ol Web Development and Native Android as well as Cross Platform Flutter Development were the new additions.",
           icon: "otter",
           link: "2021"
+        },
+         {
+          description: "Enthusiastic first yearties force COPS to start track based CSOC Dev for both frontend and backend. The Frontend track included good 'ol Web Development and Native Android as well as Cross Platform Flutter Development were the new additions.",
```
[the remark this answers is not in the document]
Comment (ItsAdityaKSingh) [reply src/pages/Index.vue:47]: Oh, sorry didn't see that. Updating right away!

File: src/pages/Index.vue (hunk 1 above)
Comment (ItsAdityaKSingh) [reply src/pages/Index.vue:48]: The general collective view of their immediate seniors XD. Your suggestion?

File: src/pages/Index.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (ItsAdityaKSingh) [reply src/pages/Index.vue]: Done!

**Review by king-11 — APPROVED** [review]

**Pull request merged**